src/main/java/com/epam/tc/hw3/voids/DifferentElementsPage.java

```diff
@@ -0,0 +1,78 @@
+package com.epam.tc.hw3.voids;
+
+import com.epam.tc.hw3.AbstractBasePage;
+import com.epam.tc.hw3.components.LogComponent;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class DifferentElementsPage extends AbstractBasePage {
```

Comment:
This page has bunch of the same elements with DifferentElementsFluentPage. Extend all common elements with come parent page.

Reply:
DifferentElementsFluentPage and DifferentElementsPage now have a common parent page.